classy/pl_modules/mixins/saving.py

```diff
@@ -22,10 +22,7 @@ def save_resources_and_update_config(
         # save examples
         source, examples = data_module.get_examples(n=5)
         experiment_folder.joinpath("data").mkdir(exist_ok=True)
-        get_data_driver(self.task, "jsonl").save(
-            examples,
-            str(experiment_folder / "data" / f"examples-{source}.jsonl"),
-        )
+        get_data_driver(self.task, "jsonl").save(examples, str(experiment_folder / "data" / f"examples-{source}.jsonl"))
```

Comment:
it seems correct to me, but double checking: here we want the gold label, right?

Reply:
Yup